root/layout/components/HeaderLogo.js

```diff
@@ -0,0 +1,20 @@
+/*
+ * @flow
+ * Copyright (C) 2018 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import React from 'react';
+
+const HeaderLogo = () => (
+  <img
+    alt="MusicBrainz"
+    className="logo"
+    src={require('../../static/images/layout/header-logo.svg')}
```

Comment:
Using `require` on an image file may look funny, since we weren't able to do this before. But Webpack has the concept of [loaders](https://webpack.js.org/concepts/loaders/) to deal with imports. We use the [file-loader](https://www.npmjs.com/package/file-loader) to handle image and CSS imports. So what this actually does is copy the image or CSS to our build dir, and return the *public URI* here. Quite nifty. You can find all of the necessary `file-loader` config is in webpack/moduleConfig.js.

For LESS files, we actually pipe them through `less-loader` (to convert them to CSS) first, then `file-loader` (to copy them and get the URI). You can see that in moduleConfig.js too.